zp-relayer/utils/constants.ts

```diff
@@ -20,6 +21,16 @@ const constants = {
   PERMIT2_CONTRACT: '0x000000000022D473030F116dDEE9F6B43aC78BA3',
   INIT_ROOT: '11469701942666298368112882412133877458305516134926649826543144744382391691533',
   MOCK_CALLDATA: '0x' + 'ff'.repeat(BASE_CALLDATA_SIZE + RAW_TX_RLP_OVERHEAD),
+  // TODO: adjust limits
+  BASE_TX_GAS: {
+    [TxType.DEPOSIT]: 650000,
```

Comment:
May we make these constants configured environment variables?

Reply:
Moved these to env parameters